service/src/main/java/com/github/defsignals/service/service/JmxService.java

```diff
@@ -0,0 +1,47 @@
+package com.github.defsignals.service.service;
+
+import org.springframework.stereotype.Service;
+
+import javax.management.*;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorFactory;
+import javax.management.remote.JMXServiceURL;
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Service
+public class JmxService {
+    public static final String URL = "service:jmx:rmi:///jndi/rmi://%s:%d/jmxrmi";
+
+    private final ConcurrentHashMap<HostPort, JMXConnector> connectionMap = new ConcurrentHashMap<>();
```

Comment:
Let's move this to a bean `JmxConnectionPool` and make the `JMXServiceURL` class the key of this connection map.